composer.json

```diff
@@ -16,10 +16,13 @@
         }
     },
     "require": {
-        "php": ">=5.6"
+        "php": ">=5.6",
+        "sebastian/comparator": "^1.1",
+        "sebastian/exporter": "^2.0",
+        "sebastian/recursion-context": "^2.0"
```

Comment:
this deps are needed to include the php 5.6 compatible versions of this packages.